api/src/LinkyLink/Controllers/HealthController.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Microsoft.AspNetCore.Mvc;

```

Comment:
Do we need copyright headers to our files?

Reply:
I don't know - Will review other OH sample apps.

Reply:
Other OH sample apps don't appear to use headers.  We'll skip for now unless we hear otherwise from someone.